tooling/bundler/src/bundle/windows/msi/wix.rs

```diff
@@ -473,41 +473,33 @@ pub fn build_wix_app_installer(
       );
     }
     WebviewInstallMode::EmbedBootstrapper { silent: _ } => {
-      let webview2_bootstrapper_path = output_path.join("MicrosoftEdgeWebview2Setup.exe");
-      std::fs::write(
-        &webview2_bootstrapper_path,
-        download(WEBVIEW2_BOOTSTRAPPER_URL)?,
-      )?;
+      let (guid, filename) = webview2_guid_path(WEBVIEW2_BOOTSTRAPPER_URL)?;
```

Comment:
this logic of downloading the installers seems to be repeated between nsis, and wix, let's move it into a function in `util.rs`